src/adhocracy_aula/adhocracy_aula/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""Adhocracy extension."""
+from pyramid.config import Configurator
+
+from adhocracy_core import root_factory
+
+
+def includeme(config):
+    """Setup adhocracy extension."""
+    # include adhocracy_core
+    config.include('adhocracy_core')
+    # commit to allow overriding pyramid config
+    config.commit()
+    # add translations
+    config.add_translation_dirs('adhocracy_core:locale/')
```

Comment:
We also have to add the translations from the core package:
   
    config.add_translation_dirs('adhocracy_core:locale/',
                                              'adhocracy_meinberlin:locale/')